docs/index.rst

```diff
@@ -3,7 +3,11 @@ sunpy core Documentation
 ************************
 
 ``sunpy`` is a community-developed, free and open-source solar data analysis environment for Python.
+It provides search and download functionality, data containers for image and time series data, as well as commonly used coordinate frames and transformations between such frames.
 
+.. image:: tour.svg
+  :width: 700
```

Comment:
Do we need to add this? I just wonder how this will work on smaller displays.

Reply:
I think it'd be fine with 300px too, I just checked on my local setup

Reply:
What if we just remove this?

Reply:
Yeah I think that'll work too